client/src/Components/Results.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+export default function Results(props) {
+
+	if(props.quizes){
+		return (
+			<div className='col-12'>
+				<h1>Welcome to QuizzTime</h1>
+
+				<select name="quizzez" id="quizzez" onChange={(e) => quizzChosen(e.target.value)}>
+					<option >select a quiz</option>
```

Comment:
quizzez is not right, you can change it to quizzes

Reply:
I agree. I sometimes have weird variable names because i try not to have same variable names even between components